src/Transport.ts

```diff
@@ -1249,10 +1256,13 @@ export class Transport extends EnhancedEventEmitter<TransportEvents>
 			this._pendingResumeConsumers.set(consumer.id, consumer);
 
 			// There is no Consumer resume in progress, do it now.
-			if (this._consumerResumeInProgress === false)
+			queueMicrotask(() => 
 			{
-				this.resumePendingConsumers();
-			}
+				if (this._consumerResumeInProgress === false)
```

Comment:
Here, at the top of the callback, we must check this._closed` and `throw new InvalidStateError('closed');` if closed.

Reply:
This should be done in all usages of `queueMicrotask()`, not just in this one.